src/main/java/uk/gov/hmcts/darts/audio/service/impl/AudioTransformationServiceImpl.java

```diff
@@ -339,19 +341,38 @@ private void notifyUser(MediaRequestEntity mediaRequestEntity,
             if (notificationTemplateName.equals(NotificationApi.NotificationTemplate.ERROR_PROCESSING_AUDIO.toString())) {
 
                 List<DefendantEntity> defendantList = mediaRequestEntity.getHearing().getCourtCase().getDefendantList();
+
                 List<String> defendantNames = new ArrayList<>();
-                for (DefendantEntity defendant : defendantList) {
-                    defendantNames.add(defendant.getName());
+                String defendants = NO_DEFENDANTS;
+                if (!defendantList.isEmpty()) {
+                    for (DefendantEntity defendant : defendantList) {
+                        defendantNames.add(defendant.getName());
+
+                    }
+
+                    defendants = defendantNames.stream().collect(Collectors.joining(","));
+                }
+
+                String courthouseName = mediaRequestEntity.getHearing().getCourtCase().getCourthouse().getCourthouseName();
+                String hearingDate = "";
+                if (courthouseName == null || courthouseName.isBlank()) {
+                    courthouseName = NOT_APPLICABLE;
+                }
+
+                if (mediaRequestEntity.getHearing().getHearingDate() == null) {
+                    hearingDate = NOT_APPLICABLE;
+                } else {
+                    hearingDate = String.valueOf(mediaRequestEntity.getHearing().getHearingDate());
                 }
 
-                String defendants = defendantNames.stream().collect(Collectors.joining(","));
+                DateTimeFormatter formatter = DateTimeFormatter.ofPattern("HH:mm:ss");
 
                 templateParams.put(REQUEST_ID, String.valueOf(mediaRequestEntity.getId()));
-                templateParams.put(COURTHOUSE, String.valueOf(mediaRequestEntity.getHearing().getCourtCase().getCourthouse()));
+                templateParams.put(COURTHOUSE, courthouseName);
                 templateParams.put(DEFENDANTS, defendants);
-                templateParams.put(HEARING_DATE, String.valueOf(mediaRequestEntity.getHearing().getHearingDate()));
-                templateParams.put(AUDIO_START_TIME, String.valueOf(mediaRequestEntity.getStartTime()));
-                templateParams.put(AUDIO_END_TIME, String.valueOf(mediaRequestEntity.getEndTime()));
+                templateParams.put(HEARING_DATE, hearingDate);
+                templateParams.put(AUDIO_START_TIME, String.valueOf(mediaRequestEntity.getStartTime().format(formatter)));
```

Comment:
`format` already returns String so don't need the `String.valueOf` part

Reply:
updated